cookbooks/calico/recipes/general_compute.rb

```diff
@@ -194,12 +194,13 @@
     group "bird"
     notifies :restart, "service[bird]", :delayed
 end
+
 template "/etc/bird/bird6.conf" do
     mode "0640"
     source "compute/bird6.conf.erb"
     owner "bird"
     group "bird"
-    notifies :restart, "service[bird]", :delayed
+    notifies :restart, "service[bird6]", :delayed
```

Comment:
This template needs to be extended to pass the 'bgp_peers' variable into it. See the block above.